src/utils/Lesson/utils.js

```diff
@@ -0,0 +1,10 @@
+export const getDeploymentAccountIdsFromLesson = lesson => {
```

Comment:
do you think it's better to have a folder for `Lesson` specifically or could this just be `lessonUtils.js`? or are you folloiwng jason's structure here and that's why it's structured this way

Reply:
also i think angela used your code for populating the `deploymentNameMap` so we could move that into here too

Reply:
I followed jason's method for this but we can change it if that's better, we'll probably move it into his utils file eventually too so not sure if this is too necessary to pin down exactly but that's a simple change